src/components/Common/Modal/ViewFollowModal/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { useEffect, useState } from 'react';
+import { useRecoilValue } from 'recoil';
+
+import Avatar from '@/components/Common/Avatar';
+import { DEFAULT_PROFILE_IMAGE } from '@/constants/profile';
+import { userAtom } from '@/recoil/user';
+import { getUserDetailsById } from '@/services/Follow/follow';
+
+import {
+  FollowButton,
+  FollowButtonWrapper,
+  NoListMessage,
+  UserList,
+  UserListItem,
+  UserName,
+  ViewFollowWrapper,
+} from './style';
+
+export default function ViewFollowModal() {
+  const user = useRecoilValue(userAtom);
+  const [activeList, setActiveList] = useState('팔로잉');
+  const [userDetails, setUserDetails] = useState([]);
+
+  useEffect(() => {
+    async function fetchUserDetails(ids: string[]) {
+      const details = await Promise.all(
+        ids.map((id: string) => getUserDetailsById(id)),
+      );
+      setUserDetails(details);
+    }
+
+    const ids =
+      activeList === '팔로잉'
+        ? user?.following.map((following) => following.user)
+        : user?.followers.map((follower) => follower.follower);
+
+    ids && ids.length > 0 ? fetchUserDetails(ids) : setUserDetails([]);
+  }, [activeList, user]);
+
+  return (
+    <ViewFollowWrapper>
+      <FollowButtonWrapper>
+        {['팔로잉', '팔로워'].map((list) => (
+          <FollowButton
+            key={list}
+            active={activeList === list}
+            onClick={() => setActiveList(list)}>
+            {list}
+          </FollowButton>
+        ))}
+      </FollowButtonWrapper>
+      {userDetails.length > 0 ? (
+        <UserList>
+          {userDetails.map((detail, index) => (
+            <UserListItem key={index}>
+              <Avatar
+                imageUrl={detail.image || DEFAULT_PROFILE_IMAGE}
+                size="small"
+              />
+              <UserName>{detail.fullName}</UserName>
+            </UserListItem>
```

Comment:
내가 팔로우/팔로잉 하는 유저의 후기도 궁금할 것 같아서
유저를 클릭했을 때 해당 유저의 프로필로 이동시켜주는건 어떨까요? 😊

`/profile/유저ID` 요 path로 이동시켜주고 모달을 닫아주면 구현이 될 것 같아유~!

```ts
const handleItemClick = (userId: string) => {
    navigate(`/profile/${userId}`);
    closeModal({ type: ModalType.VIEW_FOLLOW });
};
```

```suggestion
            <UserListItem
                key={index}
                onClick={() => handleItemClick(detail._id)}>
            >
              <Avatar
                imageUrl={detail.image || DEFAULT_PROFILE_IMAGE}
                size="small"
              />
              <UserName>{detail.fullName}</UserName>
            </UserListItem>
```

Reply:
헉 그렇게 하면 정말 센스있겠네요 !! 구현해보겠습니다 😆👍🏻